lib/Index/Index.cpp

```diff
@@ -825,8 +826,9 @@ static bool isTestCandidate(ValueDecl *D) {
 
   // 5. ...is of at least 'internal' accessibility (unless we can use
   //    Objective-C reflection)...
-#if SWIFT_OBJC_INTEROP
-  if (D->getFormalAccess() < Accessibility::Internal)
+#if !SWIFT_OBJC_INTEROP
```

Comment:
That's a great point! I'll do some digging and try to determine how to get this data at runtime.